mmseg/models/necks/jpu.py

```diff
@@ -0,0 +1,161 @@
+        self.align_corners = align_corners
+        self.out_indices = out_indices
+
+        # Note: Names of operations below are referenced from original paper.
+        self.conv3 = nn.Sequential(
+            ConvModule(
+                self.in_channels[-3],
+                self.out_channels,
+                kernel_size=3,
+                padding=1,
+                bias=False,
+                conv_cfg=conv_cfg,
+                norm_cfg=norm_cfg,
+                act_cfg=act_cfg))
+        self.conv4 = nn.Sequential(
+            ConvModule(
+                self.in_channels[-2],
+                self.out_channels,
+                kernel_size=3,
+                padding=1,
+                bias=False,
+                conv_cfg=conv_cfg,
+                norm_cfg=norm_cfg,
+                act_cfg=act_cfg))
+        self.conv5 = nn.Sequential(
+            ConvModule(
+                self.in_channels[-1],
+                self.out_channels,
+                kernel_size=3,
+                padding=1,
+                bias=False,
+                conv_cfg=conv_cfg,
+                norm_cfg=norm_cfg,
+                act_cfg=act_cfg))
+        self.dilation1 = nn.Sequential(
+            DepthwiseSeparableConvModule(
+                in_channels=3 * self.out_channels,
+                out_channels=self.out_channels,
+                kernel_size=3,
+                stride=1,
+                padding=1,
+                dilation=1,
+                dw_norm_cfg=norm_cfg,
+                dw_act_cfg=None,
+                pw_norm_cfg=norm_cfg,
+                pw_act_cfg=act_cfg))
+        self.dilation2 = nn.Sequential(
+            DepthwiseSeparableConvModule(
+                in_channels=3 * self.out_channels,
+                out_channels=self.out_channels,
+                kernel_size=3,
+                stride=1,
+                padding=2,
+                dilation=2,
+                dw_norm_cfg=norm_cfg,
+                dw_act_cfg=None,
+                pw_norm_cfg=norm_cfg,
+                pw_act_cfg=act_cfg))
+        self.dilation3 = nn.Sequential(
+            DepthwiseSeparableConvModule(
+                in_channels=3 * self.out_channels,
+                out_channels=self.out_channels,
+                kernel_size=3,
+                stride=1,
+                padding=4,
+                dilation=4,
+                dw_norm_cfg=norm_cfg,
+                dw_act_cfg=None,
+                pw_norm_cfg=norm_cfg,
+                pw_act_cfg=act_cfg))
+        self.dilation4 = nn.Sequential(
+            DepthwiseSeparableConvModule(
+                in_channels=3 * self.out_channels,
+                out_channels=self.out_channels,
+                kernel_size=3,
+                stride=1,
+                padding=8,
+                dilation=8,
+                dw_norm_cfg=norm_cfg,
+                dw_act_cfg=None,
+                pw_norm_cfg=norm_cfg,
+                pw_act_cfg=act_cfg))
```

Comment:
Can use `for` loop.

Reply:
Making dilation as an argument.
Refer to https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/decode_heads/aspp_head.py#L23.